deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -1733,6 +1733,21 @@ end}.
     end
 }.
 
+
+% ===============================
+% AWS section
+% ===============================
+
+%% @doc Whether or not to prefer EC2 IMDSv2 when querying instance metadata service.
+%% The setting defaults to true. When true, instance metadata will first be attempted using EC2 IMDSv2
+%% and will fall back to EC2 IMDSv1 upon failure.
+%% When false, EC2 IMDSv1 will be used first and no attempt will be made to EC2 IMDSv2..
+%% See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html.
+
+{mapping, "aws.prefer_imdsv2", "rabbit.aws_prefer_imdsv2",
```

Comment:
This arguably belongs to `rabbitmq_aws` which can have its own Cuttlefish schema file. Nothing in RabbitMQ uses AWS or is aware of it.

Reply:
I should perhaps clarify that RabbitMQ will discover all Cuttlefish schema files under `priv/schema` in every plugin on the code path, so I can't think of a reason why this can't be moved to `rabbitmq_aws`.

See [`rabbit_prelaunch_conf`](https://github.com/rabbitmq/rabbitmq-server/blob/master/deps/rabbit/apps/rabbitmq_prelaunch/src/rabbit_prelaunch_conf.erl#L274-L292) if you are curious how schema file discovery is performed. TL;DR: we simply scan `priv` directories of all applications on the code path :)

Reply:
Thank you for suggestion. I was afraid of having to add a new schema parser to `rabbitmq_aws` hence tried to reuse `rabbit.schema`. However, per your feedback, I've moved this AWS specific configuration flag to a new file`rabbitmq_aws.schema` in the `rabbitmq_aws/priv/schema` folder.